### DISCUSSION TITLE
Font size is so small it makes me fall asleep when i read it.

### DISCUSSION THREAD

**Discussion by Uzver123** [WerWolv/ImHex#1557] [Feedback]
Font size used across the editor is so small and there seem to be no option to increase the size. 
Only if using custom font program allows to increase font size, but then there's problem with ASCII representation becomes unreadable.

This is Arial font size set to 20

![image](https://github.com/WerWolv/ImHex/assets/143356096/c71b8ad3-0428-4b4d-b400-5167a9176d2a)

**Comment by paxcut** [comment]
The default font is the font used by the framework used to create ImHex called Dear Imgui. The font in question is a mono-spaced, pixel-perfect bitmap font which are not meant to be scaled because they lose the pixel-perfect attribute. The reason some of the characters shown in the screenshot are truncated is because the chosen custom font is either not mono-spaced or scaled to an invalid size. To avoid the issue choose a font that is mono-spaced.
